tests/inspect/inspect.go

```diff
@@ -178,6 +180,34 @@ func main() {
 		}
 	}
 
+	if globalFlags.CheckPath {
+		var idx int
+		envBytes, err := ioutil.ReadFile("/proc/self/environ")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error reading environment from \"/proc/self/environ\": %v\n", err)
+			os.Exit(1)
+		}
+		for _, v := range bytes.Split(envBytes, []byte{0}) {
+			if len(v) == 0 {
+				continue
+			}
+			if strings.Contains(string(v), "PATH=") {
+				idx = bytes.IndexByte(v, 012)
```

Comment:
Could this be `strings.Contains(string(v), "\n")`?
Since you don't use the exact index.